config.go

```diff
@@ -205,34 +206,66 @@ func (c *config) discoverInternal() error {
 		return err
 	}
 
+	internallyUsed := []string{}
+
 	for builder := range command.Builders {
+		builder := builder
 		_, found := (c.Builders)[builder]
 		if !found {
-			log.Printf("Using internal plugin for %s", builder)
-			(c.Builders)[builder] = fmt.Sprintf("%s%splugin%spacker-builder-%s",
-				packerPath, PACKERSPACE, PACKERSPACE, builder)
+			internallyUsed = append(internallyUsed, builder)
+			c.Builders[builder] = func() (packer.Builder, error) {
+				bin := fmt.Sprintf("%s%splugin%spacker-builder-%s",
+					packerPath, PACKERSPACE, PACKERSPACE, builder)
+				return c.pluginClient(bin).Builder()
+			}
 		}
 	}
+	sort.Strings(internallyUsed)
+	log.Printf("Using internal plugin for builders %v", internallyUsed)
```

Comment:
So to reduce noise I changes this logger which will now say in one line the list of internal builders ( instead of one per line ):
ex:
> Using internal plugin for builders [alicloud-ecs amazon-chroot amazon-ebs amazon-ebssurrogate amazon-ebsvolume amazon-instance azure-arm azure-chroot cloudstack digitalocean docker file googlecompute hcloud hyperone hyperv-iso hyperv-vmcx jdcloud linode lxc lxd ncloud null oneandone openstack oracle-classic oracle-oci osc-bsu osc-bsusurrogate osc-bsuvolume osc-chroot parallels-iso parallels-pvm profitbricks proxmox qemu scaleway tencentcloud-cvm triton ucloud-uhost vagrant virtualbox-iso virtualbox-ovf virtualbox-vm vmware-iso vmware-vmx yandex]